data.sql

```diff
@@ -5,4 +5,32 @@ INSERT INTO animals (name,date_of_birth, escape_attempts, neutered, weight_kg) V
  ('Charmander', '2022-2-8', 0, false, -11), ('Plantmon', '2021-11-15', 2, true, -5.7),
  ('Squirtle', '1993-4-2', 3, false, -12.13), ('Angemon', '2005-6-12', 1,true, -45),
   ('Boarmon', '2005-6-7',7,true, 20.4), ('Blossom', '1998-10-13', 3, true, 17), 
-  ('Ditto', '2022-5-14', 4, true, 22);
\ No newline at end of file
+  ('Ditto', '2022-5-14', 4, true, 22);
+
+ INSERT INTO owners(full_name, age) VALUES('Sam Smith', 34), ('Jennifer orwell',19 ), ('Bob', 45), ('Melody Pond', 77), ('Dean Winchester', 14), ('Jodie Whittakar', 38 );
+
+ INSERT INTO species(name) VALUES('Digimon'), ('Pokemon');
+
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon') WHERE name LIKE '%mon';
+
+ UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') WHERE species_id IS NULL;
+
+ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Sam Smith') WHERE name = 'Agumon';
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer orwell') WHERE name = 'Gabumon';
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer orwell') WHERE name = 'Pikachu';
+
+UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') WHERE name = 'Devimon';
+
+ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') WHERE name = 'Plantmon';
+
+vet_clinic=# UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name = 'Charmander';
+
+vet_clinic=# UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name = 'Squirtle';
+
+vet_clinic=# UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name = 'Blossom';
+
+vet_clinic=# UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') WHERE name = 'Angemon';
+
+vet_clinic=# UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') WHERE name = 'Boarmon';
```

Comment:
- [ ] Good job adding data to the database 👏 👏 . I've noticed that for owners that have more than one animal you have created more than one update 😄 , Instead of writing many `Update` for one animal owner it could be better to write one as follows : 

```sql
UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
   WHERE name = 'Gabumon' OR name = 'Pikachu';
```

![Capture](https://github.com/nilab-khwaja/Vet_Clinic_DB/assets/80458199/dc2f256b-e7f2-497f-b75a-66b55eb4fafd)